chain/starknet/src/data_source.rs

```diff
@@ -9,6 +9,8 @@ use graph::{
 use std::sync::Arc;
 
 use crate::{chain::Chain, codec, trigger::StarknetTrigger};
+use serde::de;
+use starknet::core::utils::get_selector_from_name;
```

Comment:
Please be consistent. In other files `use` statements for any external crates are grouped and separated from `crate` lines.